README.md

```diff
@@ -8,6 +8,7 @@ You're working on your Tunr app and you encounter this error. What does it mean
 
 ```text
 Your answer...
+This means templates such as articles/new templates are missing and cannot inherit controllers such as Application controllers.
```

Comment:
There is a specific template indicated.  Which file is missing, from what directory?  
This runtime error is not related to the ability to inherit from ApplicationController.  Actually, this error is raised because this controller did inherit functionality from ApplicationController, allowing ti do raise an error when the expected template was not available.